Library/Homebrew/cask/cask_loader.rb

```diff
@@ -56,24 +56,21 @@ class FromContentLoader < AbstractContentLoader
           .returns(T.nilable(T.attached_class))
       }
       def self.try_new(ref, warn: false)
-        case ref
-        when Cask, URI::Generic
```

Comment:
Slight update to https://github.com/Homebrew/brew/pull/18019/files now that URI::Generic [supports](https://github.com/sorbet/sorbet/pull/8093) `to_str`